flyteidl/protos/flyteidl/plugins/spark.proto

```diff
@@ -31,4 +33,8 @@ message SparkJob {
     // Domain name of your deployment. Use the form <account>.cloud.databricks.com.
     // This instance name can be set in either flytepropeller or flytekit.
     string databricksInstance = 9;
+    // The driver spec, used in place of the task's pod template
+    RoleSpec driverSpec = 10;
+    // The executor spec, used in place of the task's pod template
+    RoleSpec executorSpec = 11;
```

Comment:
Is there every a scenario where both of these differ from the values set in the task decorator?

Reply:
Curious what you mean here. With this change we will fail
```
@task(
  pod_template=...
)
```
in favor of specifying one or both values in
```
@task(
  task_config=Spark(
    executor_pod_template=...
    driver_pod_template=...
  )
)
```